omise/libraries/omise-plugin/helpers/charge.php

```diff
@@ -2,25 +2,73 @@
 if (! class_exists('OmisePluginHelperCharge')) {
     class OmisePluginHelperCharge
     {
+
+        private static
+            $currencyDecimals = [
+                "BIF" => 0,
+                "CLP" => 0,
+                "DJF" => 0,
+                "GNF" => 0,
+                "ISK" => 0,
+                "JPY" => 0,
+                "KMF" => 0,
+                "KRW" => 0,
+                "PYG" => 0,
+                "RWF" => 0,
+                "UGX" => 0,
+                "UYI" => 0,
+                "VND" => 0,
```

Comment:
why all those currencies are listed since they are not supported?

Reply:
The list does not relate to supported currencies. It is used in a generic function to get the number of subunits for a given currency. I just added those so the function is very future proof

Reply:
why no `PLN` polish złoty, why no Russian Rubel? But `Uganda Shilling` is there?

Reply:
Explained in the code. Doesn't need to be there if it has 2 decimal places

Reply:
ok, so what it handles all currencies in the world, right?

Reply:
Assuming the list I pulled the data from was correct - yes - at least until new currencies are added, but even then the likelihood is the new currency will probably have 2 decimal places like the majority of others

Reply:
ok... what are the chances that those currencies will be added by omise like UGX,
if you want to leave that in the code ok.
But that most likely will never, ever be used, ever...

Reply:
Yeah, I was conflicted about whether to put them all in, but eventually decided it was better to have the function work properly for as many cases as possible without being wrong, or having to return something like `null` for a currency it didn't know. You're correct though, most are edge cases that will almost certainly never be used